clang/lib/Driver/ToolChains/AMDGPU.cpp

```diff
@@ -708,6 +712,34 @@ void amdgpu::getAMDGPUTargetFeatures(const Driver &D,
                             options::OPT_m_amdgpu_Features_Group);
 }
 
+static unsigned GetFullLTOPartitions(const Driver &D, const ArgList &Args) {
+  const Arg *A = Args.getLastArg(options::OPT_flto_partitions_EQ);
+  // In the absence of an option, use the number of available threads with a cap
+  // at 16 partitions. More than 16 partitions rarely benefits code splitting
+  // and can lead to more empty/small modules each with their own overhead.
+  if (!A)
+    return std::min(16u, llvm::hardware_concurrency().compute_thread_count());
+  int Value;
```

Comment:
Even though `Value` will be initialized in `getAsInteger`, I'd still just initialize it.